src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToPublishANetCoreAppForTelemetry.cs

```diff
@@ -109,6 +109,11 @@ public void It_collects_Aot_publishing_properties(string targetFramework)
             // NativeAOT compilation requires PublishTrimmed and will be set to true if not set by the user
             var rid = EnvironmentInfo.GetCompatibleRid(targetFramework);
             var testProject = CreateTestProject(targetFramework, "AotProject", aot: true);
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                // Workaround for https://github.com/dotnet/runtime/issues/75468. Delete once the issue is fixed.
+                testProject.AdditionalProperties["LinkStandardCPlusPlusLibrary"] = "true";
```

Comment:
Can we put this in the targets file instead? That way it will actually work, instead of just passing this test.